akka-actor-typed-tests/src/test/scala/akka/actor/typed/WatchSpec.scala

```diff
@@ -3,63 +3,68 @@
  */
 package akka.actor.typed
 
+import akka.actor.typed.scaladsl.Actor
+
 import scala.concurrent._
-import scala.concurrent.duration._
-import akka.actor.typed.scaladsl.Actor._
-import akka.actor.typed.scaladsl.AskPattern._
+import akka.testkit.typed.TestKit
 
-class WatchSpec extends TypedSpec {
+object WatchSpec {
+  case object Stop
 
-  "Actor monitoring" must {
+  val terminatorBehavior =
+    Actor.immutable[Stop.type] {
+      case (_, `Stop`) ⇒ Actor.stopped
```

Comment:
Not introduced in this PR, but backticks aren't needed here since `Stop` starts with a capital S

Reply:
fixed